internal/transport/keepalive_test.go

```diff
@@ -350,6 +350,53 @@ func TestKeepaliveClientStaysHealthyWithResponsiveServer(t *testing.T) {
 	}
 }
 
+// TestKeepaliveClientFrequency creates a server which expects at most 2 client
+// pings for every 2.1 seconds, while the client is configured to send a ping
```

Comment:
This text needs to be updated in light of the number changes.

Reply:
Done.